server/service/src/sync/translations/report.rs

```diff
@@ -16,7 +20,10 @@ impl SyncTranslation for OmReportTranslator {
         "om_report"
     }
     fn pull_dependencies(&self) -> Vec<&str> {
-        vec![]
+        vec![
+            FormSchemaTranslation.table_name(),
+            OmFormSchemaTranslation.table_name(),
+        ]
```

Comment:
Added these to dependancies for report. I'm not sure they are strictly dependancies for the sync?